closed/OpenJ9.gmk

```diff
@@ -378,6 +381,7 @@ run-preprocessors-j9 : stage-j9
 			JAVA_HOME=$(BOOT_JDK) \
 			JAVA_VERSION=80 \
 			OMR_DIR=$(OPENJ9_VM_BUILD_DIR)/omr \
+			OPENJ9_BUILD=true \
```

Comment:
Why is this added? I don't see any dependency on that macro in `buildtools.mk`.

Reply:
I thought I had traced this back to a previous PR as being required, but retesting now show that its not needed.